src/optionals/math.c

```diff
@@ -221,6 +221,92 @@ static Value tanNative(DictuVM *vm, int argCount, Value *args) {
     return NUMBER_VAL(tan(AS_NUMBER(args[0])));
 }
 
+static long long gcd(long long a, long long b) {
+    long long r;
+    while (b > 0) {
+        r = a % b;
+        a = b;
+        b = r;
+    }
+    return a;
+}
+
+static Value gcdNative(DictuVM *vm, int argCount, Value *args) {
+    if (argCount == 1 && IS_LIST(args[0])) {
```

Comment:
This is quite smart, however, I wonder how it'll play out.
```
>>> Math.lcm([2222]);
Error: File 'repl', line 1
       lcm() requires 2 or more argument (1 given).
```

I know that to determine the lowest common multiple we need at least two numbers, however, do you think it'll confuse people with the fact that it's using a list of length > 2 to contextually mean more than one argument? Even though we're technically only passing a single argument (the list). Would love to hear your thoughts on this!